Fix: Replaced the change location image

jamespearson opened the pull request:
Fixes: #82 

Replaced the previous change location icon (with text) with the new one from the issue (without text)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/amaurymartiny/shoot-i-smoke?pullRequest=232) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/amaurymartiny/shoot-i-smoke?pullRequest=232) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/amaurymartiny/shoot-i-smoke?pullRequest=232) it.</sub>

amaurymartiny commented:
Thanks for the PR @jamespearson! In order to merge this PR and not break the current design, would you mind adding the text "Change Location" back, as a Text node?

font familiy: https://github.com/amaurymartiny/shoot-i-smoke/blob/master/App/util/theme.ts#L24
font color: https://github.com/amaurymartiny/shoot-i-smoke/blob/master/App/util/theme.ts#L27
font size: not sure, maybe `scale(10)`? To be played with

Thanks!

jamespearson commented:
Changed the Change Location button to a separate component.
Added Spanish and French translations

Before:
![image](https://user-images.githubusercontent.com/382457/65895565-93c75780-e3a3-11e9-83f8-e8cd10223b78.png)

After:
![image](https://user-images.githubusercontent.com/382457/65895531-8a3def80-e3a3-11e9-9a3a-9a6ccf4add29.png)
